codex-rs/app-server-protocol/src/protocol/v2.rs

```diff
@@ -348,6 +348,11 @@ pub struct ThreadCompactResponse {}
 #[ts(export_to = "v2/")]
 pub struct Thread {
     pub id: String,
+    /// The first user message in the thread, if available.
```

Comment:
Should we say "usually the first user message in the thread" so we leave open the possibility that we apply a summarizer at some point?